CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 IMPROVEMENTS:
 * Add flags `-log-level`, `-log-json` to all subcommands to control log level and json formatting. [[GH-523](https://github.com/hashicorp/consul-k8s/pull/523)]
+* Execute consul clients and servers using the docker entrypoint to keep consistent. [[GH-590](https://github.com/hashicorp/consul-k8s/pull/590)]
```

Comment:
Nit: "Consul" instead of "consul" and "Docker" instead of "docker" to be consistent with previous Changelog items.

Reply:
Nice catch!